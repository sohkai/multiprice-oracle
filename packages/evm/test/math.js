const { ethers } = require('hardhat')

function toBn(num) {
  return ethers.BigNumber.from(num)
}

module.exports = {
  toBn,
}
