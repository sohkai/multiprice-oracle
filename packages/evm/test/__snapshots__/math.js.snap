// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snx -> usdc matches query snapshot 1`] = `
Object {
  "cl": "72918452450",
  "clBuf": "72918452450",
  "sushiSpot": "72990359051",
  "uniV2Spot": "72930575038",
  "uniV3Spot": "73434657650",
  "uniV3Twap": "73435407492",
  "value": "72918452450",
}
`;
